### ISSUE TITLE
Migrate Drupal Website to a static website

### ISSUE THREAD

**Issue by luke-john** [plugorgau/plugorgau.github.io#1]
# Brief description

At the plug committee meeting on 17/02/14 there was discussion of migrating the current Drupal Plug website to a static website that could be hosted on github.
## Expected Outcomes

It should be
- easy for people to update
- easy to improve.
## Further Reading

https://help.github.com/articles/using-jekyll-with-pages

**Comment by techman83** [comment]
Should we utilise Jekyll so we can have some form of templating? There is a rich set of plugins for it.

**Comment by euandekock** [comment]
If we are rewriting to get rid of a dependency on Drupal, shouldn't we be a bit wary of implementing the new site with template dependencies built in.

**Comment by techman83** [comment]
I'm not actually so worried about template dependencies, the issue with drupal is that we don't really utilise the power of it and to fully utilise it's power is a non trivial task. Getting the site updated currently requires an account, knowing html and also requires regular maintenance to ensure it's secure.

What we get out of moving to something hosted on github is that anyone contribute to it, lowering the barrier to entry. Simple templating with simple markup lowers that further.

Github uses Jekyll for templating websites, we can use it natively or we can use Jekyll and upload a static site (the latter probably gives us a lot more flexibility).

My own site is done this way, it's pretty trivial and really easy for collaboration:

https://github.com/techman83/techman83.github.io

**Comment by luke-john** [comment]
As I see it we currently have the following content on the website

```
Welcome to PLUG

Sponsors
  PLUG Sponsors
  Hall of Fame

Get Involved
  PLUG Mailing List
  Membership & Donations

About PLUG
  Committee
  Previous Committees
  Constitution
    PLUG Constitution 2010
    PLUG Constitution 2010
    Current Constitution
    PLUG Constitution 2012
    Important PLUG Documents
    Proposed Constitution 2012 Notes
  PLUG Documents and records

Services
  Newsletters
  PLUG Shell Accounts

Events
  other
    Software Freedom Day 2012
    PLUG Logo Competition
    2011 PLUG Quiz Night
  monthly meetings
    2002 Events
    2003 Events
    2004 Events
    2005 Events
    2006 Events
    2007 Events
    2008 Events
    2009 Events
    2010 Events
    2011 Events
    2012 Events
  Annual General Meetings
    PLUG Annual General Meeting 2002
    PLUG Annual General meeting 2003
    Annual General Meeting Report 2004
    PLUG Annual General Meeting, 2005
    PLUG Annual General Meeting, 2006
    PLUG Annual General Meeting, 2007
    PLUG Annual General Meeting, 2008
    PLUG Annual General Meeting, 2009
    PLUG Annual General Meeting, 2009
    AGM Minutes 2010
    PLUG Annual General meeting, 2011

Projects
  Video Streaming Project
    Setup Procedure
    Physical Equipment
    Video Streaming: Architectural Overview
  Linux.conf.au Bid
  User Group membership manager


Other Linux-related groups
Getting Linux in Perth
Branding and Marketing Materials
Other Groups & Event
Links
```

I like the idea of having the site use a very basic templating system because it gives us the option of having some kind of integration with ugmm down the track which may encourage less technical groups to use ugmm.

**Comment by luke-john** [comment]
Closed in favor of continuing discussion at #4
